controller/PopupLimitsController.java

```diff
@@ -9,7 +9,15 @@
 import javafx.scene.control.Label;
 import javafx.scene.control.TextField;
 
-
+/**
+ * Program to open a new popup window for user interaction.
+ * Warns about imminent double entry for working day record which results in an
+ * overwrite of existing record. Requires user decision how to proceed.
+ * 
```

Comment:
this is wrong 
this part is for workinglimits 
popup to add comment why worked so early / late